woocommerce/woocommerce
# [Experimental] Product Filters Redesign: Rename individual inner filter blocks to "Filter Options"

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





Closes #49577 .



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Navigate to Product Catalog template (Appearance > Editor > Templates > Product Catalog template - or any archive template).
2. Click on "Edit" to open the WordPress editor.
3. In the editor, you should see a block inserter icon or an option to "Add Block". Click on it.
4. Search for "Product Filters (Experimental)" in the block inserter or browse the available blocks until you find them. Add it to your content.
5. In the list view, open all of the individual filter blocks.
6. Make sure you see "Filter Options" and the icon matches the design.

| Before | After |
|--------|--------|
| ![CleanShot 2024-07-16 at 14 18 14@2x](https://github.com/user-attachments/assets/5249d703-a77b-41b1-9ea7-e8434a1c72df) | ![CleanShot 2024-07-16 at 14 17 05@2x](https://github.com/user-attachments/assets/7de1c8e9-185d-46c4-a4a5-0ec9e1b11e42) | 



### Changelog entry







<details>

<summary>Changelog Entry Details</summary>

#### Significance




#### Type




#### Message 

</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 
Rename the individual filter blocks to Filter Options to match the design.

</details>
